### ISSUE TITLE
Shows User rejected request error for missing api key 

### ISSUE THREAD

**Issue by PovedaAqui** [EveripediaNetwork/wagmi-magic-connector#50]
**Describe the bug**
When I click on Magic button (within the RainbowKit Connector), I get

```
UserRejectedRequestError: User rejected request
    at MagicConnectConnector.connect (magicConnectConnector.js?97d5:72:1)
    at async connect (chunk-55IO54NW.js?40a3:727:1)

```
**To Reproduce**

```
"wagmi": "^0.11.0"
"ethers": "^5.7.2",
"@rainbow-me/rainbowkit": "^0.8.1",
"@everipedia/wagmi-magic-connector": "^0.9.1",
"next": "^13.1.5",
```
```
import { MagicConnectConnector } from "@everipedia/wagmi-magic-connector";

export const rainbowMagicConnector = ({ chains }) => ({
    id: 'magic',
    name: 'Magic',
    iconUrl: 'https://svgshare.com/i/iJK.svg',
    iconBackground: '#fff',
    createConnector: () => {
      const connector = new MagicConnectConnector({
        chains: chains,
        options: {
          apiKey: process.env.MAGIC_KEY,
          magicSdkConfiguration: {
            network: {
              rpcUrl: 'https://polygon-rpc.com', // your ethereum, polygon, or optimism mainnet/testnet rpc URL
              chainId: 137,
            },
          },
          //...Other options (check out full API below)
        },
      });
      return {
        connector,
      };
    },
  });
```

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome

**Additional context**

Properly wrapped
```
return (
    <>
      <WagmiConfig client={wagmiClient}>
        <RainbowKitProvider chains={chains}>
          <NavBar />
          <Component {...pageProps} />
        </RainbowKitProvider>
      </WagmiConfig>
    </>
    )
}
```

**Comment by PovedaAqui** [comment]
It was my mistake, you should pass the apikey as string (use template literals when no tsx).

**Comment by Royal-lobster** [comment]
Ohh. may be this should throw the api key missing error instead. will add the commit.

**Comment by Royal-lobster** [comment]
This should be fixed on `@everipedia/wagmi-magic-connector@0.10.0`. Thanks for reporting. 🙏
